src/behavioural-events/event.ts

```diff
@@ -1,5 +1,5 @@
 import { camelToUnderscore } from "../helpers/camel-to-underscore";
-import { v4 as uuidv4 } from "uuid";
+import { generateUUID } from "../helpers/uuid-helper";
```

Comment:
Just importing `uuid` was causing issues in Expo Go... so I changed to use our `generateUUID` instead.